src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/PrivilegesInterceptorImpl.java

```diff
@@ -242,6 +259,10 @@ private CreateIndexRequest replaceIndex(final ActionRequest request, final Strin
                 }
             }
 
+            if (!bulkRequest.requests().isEmpty()) {
```

Comment:
do we only need to do this is bulkRequest contains an IndexRequest?

Reply:
Read your comment [below](https://github.com/opendistro-for-elasticsearch/security/pull/1058/files#diff-5a62fb072d79733fcaf865042742d6fa03c814e4ab8b6be0b48e6be1ea2c2855R301-R303) and believe that it applies for this case as well. Is it possible to place the comment little higher for understanding.